fxa-client/ffi/src/lib.rs

```diff
@@ -202,6 +213,22 @@ pub extern "C" fn fxa_get_token_server_endpoint_url(
     })
 }
 
+/// Get the url to redirect after there has been a successful connection to FxA.
+///
+/// # Safety
+///
+/// A destructor [fxa_str_free] is provided for releasing the memory for this
+/// pointer type.
+#[no_mangle]
+pub extern "C" fn fxa_get_connection_success_url(
```

Comment:
I don't see any use of this function in the Android/Swift code, we should expose it to our consumers.

Reply:
Removed it!

Reply:
I remember Android people needed that URL, so we should actually do the opposite